scripts/generate-cluster-creds.sh

```diff
@@ -89,14 +89,25 @@ EOF
     rm kubeconfig.yaml
   fi
 
-  output=$(yq e -o json '.k8s_config' /tmp/temp_k8sconfig.yaml | jq -r -M -c .)
+  yaml_creds=$(yq '.k8s_config' /tmp/temp_k8sconfig.yaml)
+  json_creds=$(yq e -o json '.k8s_config' /tmp/temp_k8sconfig.yaml | jq -r -M -c .)
+
   # NOTE: Write to ci/ files as these files will be used in ci tests! Consider looping through all files
   # in ci dir
   for APP in merlin turing
   do
-    # Write to app chart and caraml chart ci-values.yaml
-    output="$output" yq ".environmentConfigs[0] *= load(\"/tmp/temp_k8sconfig.yaml\") | .imageBuilder.k8sConfig |= strenv(output)" -i "${SCRIPT_DIR}/../charts/${APP}/ci/ci-values.yaml"
-    output="$output" yq ".${APP}.environmentConfigs[0] *= load(\"/tmp/temp_k8sconfig.yaml\") | .${APP}.imageBuilder.k8sConfig |= strenv(output)" -i "${SCRIPT_DIR}/../charts/caraml/ci/ci-values.yaml"
+    # Write to Merlin/ Turing image builder app chart values
+    if [ $APP == "merlin" ]; then
+      yaml_creds="$yaml_creds" yq ".imageBuilder.k8sConfig |= env(yaml_creds)" -i "${SCRIPT_DIR}/../charts/${APP}/ci/ci-values.yaml"
+    else
+      json_creds="$json_creds" yq ".imageBuilder.k8sConfig |= strenv(json_creds)" -i "${SCRIPT_DIR}/../charts/${APP}/ci/ci-values.yaml"
+    fi
+    # Write to Merlin/ Turing env configs app chart values
+    yq ".environmentConfigs[0] *= load(\"/tmp/temp_k8sconfig.yaml\")" -i "${SCRIPT_DIR}/../charts/${APP}/ci/ci-values.yaml"
```

Comment:
There are 2 different ways in which the cluster credentials are stored in the helm chart values file. While the Merlin API app has been refactored to expect the image building cluster's credentials to be in yaml, Turing on the other hand, has not yet been refactored to expect its image building cluster's credentials to be in the same format (i.e. it still expects it to be in `json`). 

This block ensures that the correct app chart values get the correctly formatted credentials.